plugins/outputs/prometheus_client/README.md

```diff
@@ -1,6 +1,13 @@
 # Prometheus Client Service Output Plugin
 
-This plugin starts a Prometheus Client, listening on a port defined in the
-configuration file.
+This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all metrics on `/metrics` to be polled by a Prometheus server.
 
-It exposes all metrics on `/metrics` to be polled by a Prometheus server.
+## Configuration
+
+    # Publish all metrics to /metrics for Prometheus to scrape
+	[[outputs.prometheus_client]]
```

Comment:
this should be in an unindented code block, ie, with "\`\`\`" on either side. See other plugin readmes for an example.